src/Pact/Types/Gas.hs

```diff
@@ -224,7 +226,7 @@ instance Pretty GasArgs where
     GSelect {} -> "GSelect"
     GSortFieldLookup i -> "GSortFieldLookup:" <> pretty i
     GConcatenation i j -> "GConcatenation:" <> pretty i <> colon <> pretty j
-    GTextConcatenation nChars nStrings -> "GTextConcatenation:" <> pretty nChars <> colon <> pretty nStrings
+    GTextConcatenation nChars nStrings fixupDiv -> "GTextConcatenation:" <> pretty nChars <> colon <> pretty nStrings <> colon <> pretty fixupDiv
```

Comment:
This assumes changing this `Pretty` instance doesn't participate in on-chain computations. I skimmed through the code and found no evidence it does, but worth double-checking.

Reply:
I'm not sure about this, @jmcardon ?

Reply:
It does not. Gas logs are not part of txlog outputs that get hashed.